apps/sam-design-system-site/src/app/formly/formly.routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Routes } from '@angular/router';
+import { FormlyInputComponent } from './formly-input/formly-input.component';
+
+
+export const FORMLY_COMPONENTS = [
+    FormlyInputComponent,
+];
+
+export const FORMLY_ROUTES: Routes = [
+    {
+        path: 'formly',
+        children:[{
+            path: 'formly-input' , component: FormlyInputComponent
+        }] 
+    },
```

Comment:
The app should not have the sample items directory. It should be in a sample library that should be referenced by lazy loading.